### PR TITLE
Starting assignment

### PR DESCRIPTION
**Pull request by ShaunWMusic**

### CODE REVIEW

**Comment by ShaunWMusic** [comment]
- [x] Create a new project using `yo sass-broccoli` name the project `07-portfolio`
- [x] Initialize new folder as git repository
- [x] Using `hub` create a new repository on github
- [x] Checkpoint: Commit code from HTML grouping workshop to `master branch`
- [x] Make one change to project and commit changes
- [x] Push changes to `origin` `develop` branch
- [x] Create a pull request using the `hub` CLI
- [x] **PAGE**: Skillz `public/index.html`
  - [x] **GOAL**: Top Navigation
    - [x] Name
    - [x] Skills (This should link to `/`)
    - [x] Blog (This should link to `/blog.html`)
    - [x] City (This should link to `/city.html`)
  - [x] **GOAL**: Hero
  - [ ] **GOAL**: Projects/Skillz
    - [x] Content: Screenshots (take a screen shot of thee homework assignments or lessons so far)
    - [x] Content: Reflect (write a paragraph about what you've learned for each assignment you took a screenshot of)
    - [x] Mobile Layout (Stacked Columns)
    - [x] Tablet Layout (50/50 split)
    - [x] Desktop Layout (25/75 split with larger text)
  - [ ] **GOAL**: Footer
    - [ ] Social Media Icons w/ links to social media (if you don't have one, link to something else that you can find an icon for)
    - [ ] Logo [Copy this to `public/img/logo.svg`](https://raw.githubusercontent.com/tiy-tn-fee-2016-summer/assignments/master/07-portfolio/network.svg)
    - [ ] Logo
- [ ] **PAGE**: Blog `public/blog.html`
  - [ ] **GOAL**: Top Navigation (should be able to use the same as on skillz page)
  - [ ] **GOAL**: Blog Article
    - [ ] Desktop/Tablet Grid (50/50 split)
    - [ ] Mobile Grid (stacked Columns)
    - [ ] Article
    - [ ] Content: Week Number
    - [ ] Content: Title
    - [ ] Content: Article
      - [ ] Pick 4 days or lessons so far that you liked or struggled with the most and write 2 paragraphs about them
  - [ ] **GOAL**: Footer (should be able to use the same as on skillz page)
- [ ] **PAGE**: City `public/index.html`
  - [ ] **GOAL**: Top Navigation (should be able to use the same as on skillz page)
  - [ ] **GOAL**: Pictures
    - [ ] Desktop/Tablet Grid (25/75 split)
    - [ ] Mobile Grid (stacked Columns)
    - [ ] Overlay and CSS Filter (look at https://una.im/CSSgram/)
    - [ ] https://css-tricks.com/on-object-fit-and-object-position/
  - [ ] **GOAL**: Footer (should be able to use the same as on skillz page)

File: public/blog.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  <head>
+    <meta charset="utf-8">
+    <title>Blog</title>
+    <link rel="stylesheet" href="/app.css">
+     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.css">
+  </head>
+  <body>
+    <div class="full-page">
+    <nav class="top-nav">
+      <div class="top-nav__container">
```
Comment (rtablada) [inline public/blog.html:13]: For the top-nav, you can add one more level with your existing `.container` styles to make sure that the navbar lines up with the various grid contents.

screen shot 2016-10-18 at 1 32 27 pm

File: styles/components/blog.scss [hunk 2]
```diff
@@ -0,0 +1,52 @@
+.container {
```
Comment (rtablada) [inline styles/components/blog.scss]: Watch out, you are redeclaring rules for `.container` multiple places!

This is all of the styles needed for container. The use of `.container` on the home page is doing to much and you should refer back to the https://online.theironyard.com/paths/579/units/3686/lessons/15005 lesson about the purpose of sections, grids, and containers.

File: styles/components/body-content.scss [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  padding: $gutter;
+  text-align: left;
+}
+
+.grid-container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.container {
```
Comment (rtablada) [inline styles/components/body-content.scss]: This container is doing WAY too much. Refer back to the rock solid lesson notes in TIYO.

**Review by rtablada — COMMENTED** [review]
For the blog page, look at centering the card top to the card: https://online.theironyard.com/paths/579/units/3469/lessons/13952

Also see what parts of your grid, container, and sections can be simplified to use the rock solid principals, this will really help make things easier and reduce duplication.

**Review by rtablada — COMMENTED** [review]
To the `.img-grid` images you can set a `height` value to a set of `rem`s  and then set a hard `100%` width to make sure it fills the item. Finally you can add `object-fit: cover` which will make it so that the images do not stretch.

File: public/blog.html (hunk 1 above)
Comment (ShaunWMusic) [reply public/blog.html:13]: Nav-bar updated :)

File: styles/components/blog.scss (hunk 2 above)
Comment (ShaunWMusic) [reply styles/components/blog.scss]: I deleted this selector. it actually had no purpose other than to remind me how to use containers properly.

File: styles/components/body-content.scss (hunk 3 above)
Comment (ShaunWMusic) [reply styles/components/body-content.scss]: got it. fix it. thanks 👍

**Comment by ShaunWMusic** [comment]
just added height, width and object-fit. thanks!! @rtablada

**Pull request merged**